cclib/method/volume.py

```diff
@@ -266,9 +269,10 @@ def electrondensity_spin(ccdata, volume, mocoeffslist):
     density.data = numpy.zeros(density.data.shape, "d")
 
     conversion = convertor(1, "bohr", "Angstrom")
-    x = numpy.arange(density.origin[0], density.topcorner[0] + density.spacing[0], density.spacing[0]) / conversion
-    y = numpy.arange(density.origin[1], density.topcorner[1] + density.spacing[1], density.spacing[1]) / conversion
-    z = numpy.arange(density.origin[2], density.topcorner[2] + density.spacing[2], density.spacing[2]) / conversion
+    gridendpt = density.topcorner + 0.5 * density.spacing

```

Comment:
These 4 lines you could probably put in a little helper function and pass in `density` or `wanefn` above.

Reply:
Yes. I've added a helper function for this part of the code. Thanks.